### PR TITLE
Combo Detection plus Team and Collectible Update

### PR DESCRIPTION
**Pull request by danielwiktorczyk**
Does not include the actual digging part just yet

Try it out in my scene! 

Don't worry about the commit history, it will be squashed afterward

### CODE REVIEW

File: Assets/Scripts/Tiling/Tile.cs [hunk 1]
```diff
@@ -1,3 +1,4 @@
+using Mirror.Cloud.Examples.Pong;
```
Comment (eli-lam) [inline Assets/Scripts/Tiling/Tile.cs:1]: Why is this here?

Comment (danielwiktorczyk) [reply Assets/Scripts/Tiling/Tile.cs:1]: Auto complete probably imported it haha. Will remove now

**Review by zahranikbakht — CHANGES_REQUESTED** [review]
Great job! I just found this error when I try to run my own scene on this branch. It just pops up once when I enable the player for the first time.
errorimage

File: Assets/Scripts/Combos/ComboPlayer.cs [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+    [SerializeField] private float cooldownMax;
+
+    private ComboManager comboManager;
+    private ComboPlayer[] team;
+    private float cooldownProgress;
+
+    public bool IsOnCooldown { get; private set; }
+
+    public bool IsInvolvedInACombo4;
+    public bool IsInvolvedInACombo3;
```
Comment (zahranikbakht) [inline Assets/Scripts/Combos/ComboPlayer.cs:17]: I don't see these two variables being used anywhere. Ignore this if I'm blind lol

Comment (danielwiktorczyk) [reply Assets/Scripts/Combos/ComboPlayer.cs]: Cleaned up!

**Review by zahranikbakht — APPROVED** [review]

**Review by zhengHugo — APPROVED** [review]

**Pull request merged**